packages/shell/src/Adapter/useAdapter.ts

```diff
@@ -88,7 +94,12 @@ export const useAdapter = () => {
           .then(() => successConnection(offlineConnector));
       }
     },
-    [activeChain.blockchain, dispatch, handleWalletConnectionError]
+    [
+      activeChain.blockchain,
```

Comment:
change it to just `activeChain`

Reply:
why in this one should be just activeChain if the only attribute used activeChain.blockchain?

Reply:
this property comes from the StateManagement where we are applying object restructuring and reducer patterns all the time to compute the state, I strongly recommend not doing it for avoiding issues of not refresh of the app... because we are generating new pointers of the object with every state change

Reply:
actually, using